sweagent/agent/models.py

```diff
@@ -696,10 +696,8 @@ def get_model(args: ModelArguments, commands: Optional[list[Command]] = None):
         return HumanThoughtModel(args, commands)
     if args.model_name == "replay":
         return ReplayModel(args, commands)
-    elif args.model_name.startswith("gpt") or args.model_name.startswith("ft:gpt"):
+    elif args.model_name.startswith("gpt") or args.model_name.startswith("ft:gpt") or args.model_name.startswith("azure:gpt"):
         return OpenAIModel(args, commands)
-    elif args.model_name.startswith("claude"):
```

Comment:
Why did you delete this?

Reply:
I tried to undo the changes for the model list, and seems like I copy and pasted the code from wrong commit.. Sorry, my bad